schema/spec_test.go

```diff
@@ -46,6 +54,10 @@ func TestSpecExamples(t *testing.T) {
 	}
 
 	for _, example := range examples {
+		if example.Err == errFormatInvalid { // ignore
```

Comment:
There are many examples/snippets in the spec not meant for validation.
Hmm, but you're right, we should only ignore if there is no MediaType set.